docset/winserver2022-ps/netsecurity/Disable-NetFirewallHyperVRule.md

```diff
@@ -0,0 +1,336 @@
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PolicyStoreSourceType
+Specifies that Hyper-V firewall rules that match the indicated policy store source type are disabled. 
+This parameter value is automatically generated and should not be modified. 
+The acceptable values for this parameter are:
+- Local: The object originates from the local store. 
+- MDM: The object originates from the MDM store.
+
+By default, the Local store is queried.
+
+```yaml
+Type: PolicyStoreType[]
+Parameter Sets: ByQuery
+Aliases: 
+Accepted values: None, Local, MDM
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Protocol
+Specifies that only matching Hyper-V firewall rules with the indicated Protocol are disabled.
+The acceptable values for this parameter are:
+- Protocols by number:  0-255. 
+- Protocols by name:  TCP, UDP, ICMPv4, or ICMPv6. 
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to run the cmdlet.
+If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
+The throttle limit applies only to the current cmdlet, not to the session or to the computer.
+
+```yaml
+Type: Int32
+Parameter Sets: (All)
+Aliases: 
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -VMCreatorId
+Specifies that only matching Hyper-V firewall rules with the specified VMCreatorId are disabled.
+The format for this value is a Guid enclosed in brackets, i.e '{9E288F02-CE00-4D9E-BE2B-14CE463B0298}'
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### Microsoft.Management.Infrastructure.CimInstance#root\StandardCimv2\NetFirewallHyperVRule
+The `Microsoft.Management.Infrastructure.CimInstance` object is a wrapper class that displays Windows Management Instrumentation (WMI) objects.
+The path after the pound sign (`#`) provides the namespace and class name for the underlying WMI object.

```

Comment:
Please clarify whether this should be a hash (`#`) or a pound sign (`£`). Thanks.